support-frontend/assets/pages/supporter-plus-landing/components/threeTierCards.tsx

```diff
@@ -81,17 +74,20 @@ export function ThreeTierCards({
 				}
 				return (
 					<ThreeTierCard
+						product={cardContent.product}
```

Comment:
Would passing the `cardContent` object through to `ThreeTierCard` be slightly cleaner (maybe its quite big?) versus multiple individual props?

Reply:
Good suggestion. I've just pushed a commit which moves the `CardContent` type into `threeTierCard.tsx`, and we now just pass the whole thing in